ember_debug/adapters/basic.js

```diff
@@ -113,7 +113,7 @@ export default EmberObject.extend({
   */
   connect() {
     return new Promise((resolve, reject) => {
-      $(() => {
+      document.addEventListener('DOMContentLoaded', () => {
```

Comment:
I think `DOMReady` equivalents probably needs a small helper, given that this event will not fire if the script is loaded after the DOM is ready.

It would require something like the following to handle the case:

```javascript
const subscribers = [];

function handleDOMContentLoaded() {
  document.removeEventListener('DOMContentLoaded', handleDOMContentLoaded)
  let func
  while ((func = subscribers.shift()) {
    setTimeout(func, 0)
  }
}

function scheduleDomReady(func) {
  if (document.readyState === "interactive" || document.readyState === "complete") {
    setTimeout(func, 0)
  } else {
   if (subscribers.length === 0) {
     document.addEventListener('DOMContentLoaded', handleDOMContentLoaded)
   }
    subscribers.push(func)
  }
}
```